### DISCUSSION TITLE
Is there a plan for vue?

### DISCUSSION THREAD

**Discussion by killjin** [filamentphp/filament#6924] [Ideas]
Is there a plan for vue or inertia and vue?

**Comment by danharrin** [comment]
I have experimented with it before, but decided against building a Filament version with it. I don't think it would be as good a product with the same developer experience if it used Vue, because of how Livewire components work without any build step or separate API controllers.

**Reply by killjin** [reply]
Thank you, the official paid panel of laravel can't meet our requirements at all, the filament is very good, the back end is very elegant and comprehensive, if it can customize the vue page, I think it will be better.

**Reply by killjin** [reply]
Can you give me a reference for converting vue, let's maintain a filament vue branch?
